ethereum_history_api/circuits/lib/src/rlp/types.nr

```diff
@@ -25,26 +25,61 @@ struct RlpFragment {
 }
 
 impl RlpFragment {
-    fn assert_eq_bytes<FIELD_NAME_LEN, MAX_RLP_LEN, MAX_VALUE_LEN>(
+    fn assert_eq_bytes_length<FIELD_NAME_LEN, MAX_RLP_LEN, MAX_VALUE_LEN>(
```

Comment:
We also use it when comparing logs bloom for receipts. They are an array of bytes of the known, constant length. I think it's more readable to simply give these values as arguments, instead of creating a bounded vector.

Reply:
Ok, makes sense. So we have two methods. One takes a bounded vec and another does not need length and can take it from generic arg. Seems like the one with bounded vec is the one that needs implementation as a more general one and the one for static bytes needs to just call another one with length.

Also - I would propose to not use BoundedVec in a name as it's a name of a data structure and not of the nature of data. Maybe `assert_eq_bytes` and `assert_eq_padded_bytes`?